package.json

```diff
@@ -1,60 +1,28 @@
 {
   "name": "scratch-blocks",
-  "version": "1.1.86",
+  "version": "2.0.0",
   "description": "Scratch Blocks is a library for building creative computing interfaces.",
   "author": "Massachusetts Institute of Technology",
   "license": "Apache-2.0",
-  "homepage": "https://github.com/scratchfoundation/scratch-blocks#readme",
+  "homepage": "https://github.com/gonfunko/scratch-blocks#readme",
   "repository": {
     "type": "git",
-    "url": "https://github.com/scratchfoundation/scratch-blocks.git"
+    "url": "https://github.com/gonfunko/scratch-blocks.git"
   },
-  "main": "./dist/vertical.js",
-  "browser": "./shim/vertical.js",
+  "main": "./dist/main.js",
+  "private": true,
   "scripts": {
-    "deploy": "rimraf gh-pages/closure-library/scripts/ci/CloseAdobeDialog.exe && gh-pages -t -d gh-pages -m \"Build for $(git log --pretty=format:%H -n1) [skip ci]\"",
-    "prepare": "husky install",
-    "prepublish": "python build.py && webpack",
-    "test": "npm run test:messages && npm run test:unit",
-    "test:lint": "eslint .",
-    "test:messages": "npm run translate && node i18n/test_scratch_msgs.js",
-    "test:unit": "node tests/jsunit/test_runner.js",
-    "translate": "node i18n/js_to_json.js && node i18n/json_to_js.js",
-    "translate:sync:src": "tx-push-src scratch-editor blocks msg/json/en.json",
-    "translate:sync:translations": "node i18n/sync_tx_translations.js",
-    "translate:update": "npm run translate:sync:src && npm run translate:sync:translations"
-  },
-  "dependencies": {
-    "exports-loader": "^0.7.0",
-    "google-closure-library": "^20190301.0.0",
-    "imports-loader": "^0.8.0",
-    "scratch-l10n": "^3.18.3"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack --mode production",
+    "start": "webpack serve --open --mode development"
   },
   "devDependencies": {
-    "@commitlint/cli": "17.8.1",
-    "@commitlint/config-conventional": "17.8.1",
```

Comment:
I think we probably want to keep these so that we're conforming to their commit conventions. Do you think you could also wire up the existing `commitlint` Github workflow to run on commit? I don't seem to have permissions to do that in this repo.

Reply:
Re-added commitlint, will set it up later.